test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
```

Comment:
I'm not sure I see any throttling in practice. Batches of 34 or 20 tend to complete in ~1-2sec. It's only when I throw all 680 requests at the same time (with no pauses) that I start seeing timeouts.